lib/addons/ack/serviceMappings.ts

```diff
@@ -43,7 +43,8 @@ export enum AckServiceName {
   ROUTE53RESOLVER = "route53resolver",
   CLOUDWATCH = "cloudwatch",
   ACMPCA = "acmpca",
-  CLOUDWATCHLOGS = "cloudwatchlogs"
+  CLOUDWATCHLOGS = "cloudwatchlogs",
+  KAFKA = "kafka"
```

Comment:
Minor: this list of supported services is not ordered and it makes it hard to see which services are supported. Let's order the list alphabettically.